SS2_SingleSample/STARPipeline_KCO.wdl

```diff
@@ -0,0 +1,174 @@
+task StarRun {
+  File input_fastq_read1
+  File input_fastq_read2
+  File gtf
+  File  starGenome
+
+  command {
+   tar -zxvf ${starGenome}
+   STAR  --readFilesIn ${input_fastq_read1} ${input_fastq_read2} --genomeDir ./star  --quantMode TranscriptomeSAM   --outSAMstrandField intronMotif --genomeLoad NoSharedMemory --sjdbGTFfile ${gtf} --readFilesCommand "zcat" --twopassMode Basic --outSAMtype BAM SortedByCoordinate --outSAMattributes NH HI NM MD AS XS --outSAMunmapped Within --limitBAMsortRAM 20000000000 
```

Comment:
Should any other arguments be exposed as parameters to the task? Are any of them likely to change? Also there are some extra spaces in this command.